subprojects/docs/src/docs/release/notes.md

```diff
@@ -532,7 +533,8 @@ The source path for a `JavaCompile` task should be set via the `CompileOptions.s
 ### Cannot specify `--processor-path` directly as a Java compiler arg
 
 Adding `-processorpath` or `--processor-path` to the `CompileOptions.compilerArgs` list is now prohibited.
-The annotation processor path for a `JavaCompile` task should be set via the `CompileOptions.annotationProcessorPath` property.
+Annotation processor should instead be added to the `annotationProcessor` configuration.
+If you need to override the annotation processor path for a `JavaCompile` task, you can set it via the `CompileOptions.annotationProcessorPath` property.
```

Comment:
I wouldn't even mention this one, it's rather low level and covered in the DSL reference.